PythonVisualizations/OrderedArray.py

```diff
@@ -0,0 +1,328 @@
+import random
+import time
+from tkinter import *
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+CELL_SIZE = 50
+CELL_BORDER = 2
+CELL_BORDER_COLOR = 'black'
+ARRAY_X0 = 100
+ARRAY_Y0 = 100
+FONT_SIZE = 20
+VALUE_FONT = ('Helvetica', FONT_SIZE)
```

Comment:
`VALUE_FONT` and `FONT_SIZE` are already defined in `VizualizationApp.py`. I don't think this will cause issues but it is unnecessary.

Reply:
Also, any constants unique to `OrderedArray` need to be defined as attributes of that class (to avoid having global name conflicts)